app/Commands/StartCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Commands;
+
+use App\InitializesCommands;
+use App\Services;
+use App\Shell\Docker;
+use Illuminate\Support\Str;
+use LaravelZero\Framework\Commands\Command;
+use PhpSchool\CliMenu\CliMenu;
+
+class StartCommand extends Command
+{
+
+    use InitializesCommands;
+
+    protected $signature = 'start {container?}';
```

Comment:
Would it hurt at all to make this `containerId`

Reply:
No. Makes more sense as `containerId`